### PR TITLE
refactor(frontend): cloudscape library for charts

### PR DESCRIPTION
**Pull request by denisquesada**
## Description

[Cloudscape library](https://github.com/cloudscape-design/components) assessment for charts in PDoA to meet A11y guidelines for charts as follows:
* Variable line variations are available (solid, dotted, etc) with different data sets on line charts.
* Our custom color set works with the charts.
* Accessible tooltips, content that appears on hover may be moved over with a pointer without disappearing.

## Testing

Describe how you tested your changes and/or give instructions to the reviewers on how they can verify them.

Go to https://d3dd0mmx0g50vr.cloudfront.net/another-dashboard and play with the charts.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/components/LineChartWidget.module.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+svg[role="application"] {
+    > g[role="group"]{
+        > g[role="group"]{
+            @for $i from 1 through 10 {
+                & > g[role="group"]:nth-child(#{$i}) {
+                  > path {
+                    stroke-dasharray:#{($i - 1)*5} #{($i - 1)*5};
```
Comment (mabreuortega) [inline frontend/src/components/LineChartWidget.module.scss:7]: awesome ;-)

File: frontend/src/components/LineChartWidget.tsx [hunk 2]
```diff
@@ -136,92 +126,31 @@ function LineChartWidget(props: Props) {
             )}
             {data && data.length > 0 && (
                 <div aria-hidden="true">
```
Comment (mabreuortega) [inline frontend/src/components/LineChartWidget.tsx]: Have you seen how looks the ARIA properties of the new chart?

**Review by mabreuortega — APPROVED** [review]
LGTM

File: frontend/src/components/BarChartWidget.tsx [hunk 3]
```diff
@@ -151,122 +91,32 @@ const BarChartWidget = (props: Props) => {
...
-                                                                tick,
-                                                                xAxisLargestValue,
-                                                                props.significantDigitLabels,
-                                                                props.bars.slice(1),
-                                                                props.columnsMetadata,
-                                                            );
-                                                        }}
-                                                    />
-                                                )}
-                                            {!props.hideDataLabels && !props.stackedChart && (
```
Comment (mabreuortega) [inline frontend/src/components/BarChartWidget.tsx:249]: hideDataLabels is not used here. I think something is missing

File: frontend/src/components/BarChartWidget.tsx [hunk 4]
```diff
@@ -151,122 +91,32 @@ const BarChartWidget = (props: Props) => {
...
-                                        Number(value),
-                                        xAxisLargestValue,
-                                        props.significantDigitLabels,
-                                        "",
-                                        "",
-                                        columnMetadata,
-                                    );
-                                }}
-                            />
-                            {!props.hideLegend && (
```
Comment (mabreuortega) [inline frontend/src/components/BarChartWidget.tsx:211]: hideLegend is also missing

Comment (mabreuortega) [reply frontend/src/components/BarChartWidget.tsx:211]: check showMobilePreview, In general, we don't want to break any existing contract outside of the component. Please, go over each property of the props and ensure they are used in the new chart.

**Review by mabreuortega — CHANGES_REQUESTED** [review]
Check each props to ensure they are used in the chart options

File: frontend/src/components/LineChartWidget.tsx (hunk 2 above)
Comment (denisquesada) [reply frontend/src/components/LineChartWidget.tsx]: yes, they look good.

File: frontend/src/components/BarChartWidget.tsx (hunk 3 above)
Comment (denisquesada) [reply frontend/src/components/BarChartWidget.tsx:249]: unsupported with the current library. it's being removed from the UI and component contract.

File: frontend/src/components/BarChartWidget.tsx (hunk 4 above)
Comment (denisquesada) [reply frontend/src/components/BarChartWidget.tsx:211]: `hideLegend` is a legacy property not linked to the UI. it's being removed from the component contract.

**Review by mabreuortega — APPROVED** [review]
Lgtm

**Pull request merged**